tools/PI/DevHome.PI/SettingsUi/AddToolControl.xaml.cs

```diff
@@ -39,7 +42,9 @@ private void HandleBrowseButton()
                     lpstrFilter = pFilter,
                     nFilterIndex = 1,
                     nMaxFile = 255,
-                    hwndOwner = BarWindow.ThisHwnd,
+
+                    // TODO - This should be the Settings window, not the bar window

```

Comment:
nit: can we add a work item for this?

Reply:
This is the bug 50692806 in the internal database I talked about during the sync today.

Reply:
@andreww-msft  FYI